setup.py

```diff
@@ -43,7 +43,7 @@
         'xstatic',
         'XStatic-asciinema-player',
         'xstatic-bootbox',
-        'xstatic-bootstrap<4.0.0.0',
+        'xstatic-bootstrap>=4.0.0.0,<5.0.0',
```

Comment:
please use 4 digits for both version numbers

Reply:
ouch, will fix